drivers/sensor/mcp9600/mcp9600.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022, Maxmillion McLaughlin
+ * Copyright (c) 2020, SER Consulting LLC / Steven Riedl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MCP9600_H_
+#define ZEPHYR_DRIVERS_SENSOR_MCP9600_H_
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+
+#define MCP9600_REG_TEMP_HOT 0x00
+
+#define MCP9600_REG_TEMP_DIFF 0x01
+#define MCP9600_REG_TEMP_COLD 0x02
+#define MCP9600_REG_RAW_ADC   0x03
+
+#define MCP9600_REG_STATUS 0x04
+
+#define MCP9600_REG_TC_CONFIG  0x05
+#define MCP9600_REG_DEV_CONFIG 0x06
+#define MCP9600_REG_A1_CONFIG  0x07
+
+#define MCP9600_REG_A2_CONFIG 0x08
+#define MCP9600_REG_A3_CONFIG 0x09
+#define MCP9600_REG_A4_CONFIG 0x0A
+#define MCP9600_A1_HYST	      0x0B
+
+#define MCP9600_A2_HYST	 0x0C
+#define MCP9600_A3_HYST	 0x0D
+#define MCP9600_A4_HYST	 0x0E
+#define MCP9600_A1_LIMIT 0x0F
+
+#define MCP9600_A2_LIMIT	0x10
+#define MCP9600_A3_LIMIT	0x11
+#define MCP9600_A4_LIMIT	0x12
+#define MCP9600_REG_ID_REVISION 0x13
+
+struct mcp9600_data {
+	int32_t temp;
+};
+
+struct mcp9600_config {
+	const struct i2c_dt_spec bus;
+};
```

Comment:
This isn't really worth it's own private header file. Just move these definitions to mcp9600.c